doc/adrs/XXX_add_table_description_in_view_dsl.adoc

```diff
@@ -0,0 +1,99 @@
+= ADR-XXX Add Table Description in View DSL
+
+== Context
+
+Add table description to View DSL.
+
+== Decision
+
+=== View DSL
+
+* *Name*: _view#Identifier_, name of the representation.
+* *Label Expression*: _AQL expression_, default name for an instance of this representation.
+* *Icon URLs Expression*: _AQL expression_, icons for this representation.
```

Comment:
Why? We don't have custom icons for other representations. If we need to customize the icon of the representation, it shall be done as a dedicated improvement (and thus ADR) and it should be supported by all representations with something like `RepresentationDescription#iconExpression`. I don't think we are mentioning URLs in the view DSL for images. We have `NodeLabelStyle#labelIcon` or `ImageNodeStyleDescription#shape`.